internal/warehouse/github_actions.go

```diff
@@ -138,12 +138,22 @@ func (w *warehouse) handleWorkflowRuns(ctx context.Context, owner, repo string,
 		}
 
 		w.logger.Info().Msgf("getting workflow runs for workflow %s", *workflow.Name)
+
+		if workflow.ID == nil {
+			w.logger.Debug().Msg("workflow with empty ID")
+			break
+		}
 		// we get a page of 30 workflow runs until next page is 0
 		for {
 
-			w.logger.Debug().Str("workflow", *workflow.Name).Str("ID", fmt.Sprintf("%d", *workflow.ID)).Msg("getting runs of")
+			if workflow.Name != nil {
+				w.logger.Debug().Str("workflow", *workflow.Name).Str("ID", fmt.Sprintf("%d", *workflow.ID)).Msg("getting runs of")
+			}
+
 			if workflowRunsPage, resp, err = getWorkflowsRuns(ctx, w.githubClient, owner, repo, *workflow.ID, opt); err != nil {
-				w.logger.Warn().Str("workflow", *workflow.Name).Str("ID", fmt.Sprintf("%d", *workflow.ID)).AnErr("Error", err).Msg("error occurred")
+				if workflow.Name != nil {
+					w.logger.Warn().Str("workflow", *workflow.Name).Str("ID", fmt.Sprintf("%d", *workflow.ID)).AnErr("Error", err).Msg("error occurred")
```

Comment:
```suggestion
					w.logger.Warn().Str("workflow", *workflow.Name).Str("ID", fmt.Sprintf("%d", *workflow.ID)).AnErr("error", err).Msg("error occurred")
```

we shouldn't need the `if workflow.Name != nil` here right? Since we check it right above?